### PR TITLE
Update list of projects

### PR DESCRIPTION
**Pull request by spestana**
Some minor edits to try and clarify project information.

- changed description of list of projects
- removed redundant "hackweek" text
- changed example project in the table
- added the project spreadsheet url link

### CODE REVIEW

**Review by scottyhq — APPROVED** [review]

**Comment by JessicaS11** [comment]
@spestana I don't actually see the project list url in the changes.

File: book/projects/index.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ You can find our comprehensive guide for organizing projects before, during, and
 
 ## List of Projects
 
-Here is our current list of project for our {{dates}} {{ hackweek }} hackweek:
+Projects for the {{dates}} {{ hackweek }} will be listed here during the event. Prior to them being listed here, participants can express their interest in one or more projects on the projects spreadsheet: {{ project_spreadsheet_url }}
```
Comment (spestana) [inline book/projects/index.md:16]: @JessicaS11 The "project_spreadsheet_url" is what I meant by the project list. Though this was something we used for GeoSMART last year, and haven't used this year. I'm not sure if it should still be included in the template or not.

Comment (JessicaS11) [inline book/projects/index.md:16]:
```suggestion
Projects for the {{dates}} {{ hackweek }} will be listed here during the event. Prior to them being listed here, participants can express their interest in one or more projects on the projects spreadsheet: {{ project_spreadsheet_url }}.
```

Comment (JessicaS11) [reply book/projects/index.md:16]: > I'm not sure if it should still be included in the template or not.

Sounds like a good discussion topic for a future hw-as-a-service meeting!

**Comment by JessicaS11** [comment]
Not sure why the CI is failing - I seem to recall this error appeared in the SnowEx book at some point. @scottyhq @jomey was there a good fix for this one?

`/home/runner/work/jupyterbook-template/jupyterbook-template/book/projects/index.md:5: WARNING: 'myst' cross-reference target not found: '' [myst.xref_missing]`

**Comment by scottyhq** [comment]
> /home/runner/work/jupyterbook-template/jupyterbook-template/book/projects/index.md:5: WARNING: 'myst' cross-reference target not found: '' [myst.xref_missing] 

It's strange that index.md:5 suggests to me that the problem is on line 5, but i've never quite understood the jupyterbook warning and error line numbers... In any case I made a suggestion above - the error is coming from an empty link to an example project

**Pull request merged**